src/w3c/abstract.js

```diff
@@ -4,10 +4,10 @@ import { pub } from "core/pubsubhub";
 import { l10n, lang } from "core/l10n";
 export const name = "w3c/abstract";
 
-export async function run(conf) {
+export async function run() {
   const abs = document.getElementById("abstract");
   if (!abs) {
-    pub("error", `Document must have one element with \`id="abstract"`);
+    pub("error", "Document must have one element with `id=\"abstract\".");
```

Comment:
It looks like there is a nit in the master branch. `` `id="abstract" `` vs `` `id="abstract"` ``